### DISCUSSION TITLE
2021-10-13 Weekly Meeting 25 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#106] [Weekly Meetings]
Weekly Meeting 25 is happening on Wednesday October 13, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - Let's review our vision! #72 
   - OMI Website
   - glTF subgroup
   - IPSME RFP https://github.com/omigroup/omigroup/issues/107
   - [OMI Group Project Ideas](https://github.com/omigroup/omigroup/discussions/105)
   - Event proposals for the Fall/Early Winter

Add ons: 
   - As you see things of interest in discord, a convo or a demo, take a screenshot and drop it into the comments of the next agenda.

**Comment by humbletim** [comment]
- [OMI Group Project Ideas - Nontechnical Discussion #105](https://github.com/omigroup/omigroup/discussions/105) @Elirudite

**Reply by mrmetaverse** [reply]
Added this new Req for participation https://github.com/omigroup/omigroup/issues/107

**Comment by mrmetaverse** [comment]
start attendance: 
![image](https://user-images.githubusercontent.com/63426722/137171517-929f2208-afc8-4d62-b5f5-2f9447fb89f0.png)

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/omigroup/issues/72 - added link for others to participate in convo async

**Comment by mrmetaverse** [comment]
Website group did not meet yesterday. Next meeting is slated for Tuesday 10/19/21 at 1pm EDT

**Comment by mrmetaverse** [comment]
gLTF group notes: There is a speaker object inside of blender, so we can use that for blender audio emitting! On the godot side of things, the gltf extensions api is in the unstable release of godot which will allow us to build the godot version of the [omi audio emitter](https://github.com/omigroup/gltf-extensions/pull/1) extension, exploring some the import API for importing gltf content into blender and translating it into blender. SO if there is an audio emtter inside of a gLTF file, you could bring that into blender. 

A [4 stage proposal process](https://github.com/omigroup/gltf-extensions/issues/21) was proposed. A vote is slated to occur this week.

**Comment by mrmetaverse** [comment]
Dr. Nevelsteen has submitted a [Call for Participation](https://github.com/omigroup/omigroup/issues/107) requesting help creating interoperable integrations. 

IPSME is a very interesting concept, that is fairly easy to integrate in existing systems. The network effects are massive, as more people integrate pieces of their applications together. If you have two systems you want to integrate, you're the target market for supporting this effort. 

Nova also had a demo on demo night using a dbus system to implement one of these pubsub messaging processes. 

voiced Concerns: Privacy, and actual implementation. But those are things we would discuss in the working group.

**Comment by mrmetaverse** [comment]
OMI group project ideas: There are a ton of really cool ideas for how OMI can continue to grow and add value. https://github.com/omigroup/omigroup/discussions/105

Do we want to run a hackathon? 
Can we build on the mapping ideas of mapping the interoperability of the metaverse?

**Comment by mrmetaverse** [comment]
Do we want to create a documentation site that showcases projects and how they pertain to interoperability. Explicitly visualizing the mapping of how pieces of the metaverse are connected is also of interest.

**Comment by mrmetaverse** [comment]
Global XR conference reached out asking if we want to put together a panel
